cogs/annual_handover_and_reset.py

```diff
@@ -62,7 +61,7 @@ async def committee_handover(self, ctx: TeXBotApplicationContext) -> None:
             await initial_response.edit(
                 content=(
                     ":warning: This command requires TeX-Bot to hold a role higher than "
-                    "that of the \"Committee\" role to perform this action. Operation aborted."
+                    'that of the "Committee" role to perform this action. Operation aborted.'
```

Comment:
Prefer strings to use double quotes & escape interior double quotes rather than using single quotes